l10n-pocket/en/brands.ftl

```diff
@@ -34,10 +40,16 @@
 ## Third parties
```

Comment:
Suggestion (non-blocking): not introduced in this PR, but I'm not sure it makes sense to have a grouping called `Third parties`? Most of these could be under `Company names` which already exists at the top. Things like `Apple App Store` should be under `Apple Products` etc.